rahuldkjain/github-profile-readme-generator
# feature: add hashnode to socials input and preview

closes #452



## What type of PR is this? (check all applicable)


## Description

Add inputs and outputs for the hashnode social media link

## Related Tickets & Documents

n/a

## QA Instructions, Screenshots, Recordings

1. Scroll to the bottom of the Socials inputs and find the hashnode placeholder
<img width="1123" alt="Screen Shot 2021-10-31 at 11 21 41 PM" src="https://user-images.githubusercontent.com/19453294/139621259-88d65e70-dd37-4dc8-ad28-851c23461238.png">
2. Fill in "@ username"
<img width="639" alt="Screen Shot 2021-10-31 at 11 17 51 PM" src="https://user-images.githubusercontent.com/19453294/139621469-712a31a9-e38d-439e-8b58-1d4e0e6207f2.png">
3. See preview (note: svg has been added in this branch)
<img width="907" alt="Screen Shot 2021-10-31 at 11 23 28 PM" src="https://user-images.githubusercontent.com/19453294/139621472-57ce0595-5b4b-48ad-aa41-242fa560447c.png">
4. See html elements in markdown
<img width="1332" alt="Screen Shot 2021-10-31 at 11 22 59 PM" src="https://user-images.githubusercontent.com/19453294/139621474-e328d4e3-3851-4f30-9e10-65e2fbc57d3f.png">
5. See link working
<img width="886" alt="Screen Shot 2021-10-31 at 11 36 54 PM" src="https://user-images.githubusercontent.com/19453294/139622310-53dcb32f-fd3e-4ffb-a197-e0ca9b431420.png">



## Added to documentation?
